api/batch/src/main/java/it/gov/pagopa/rtd/transaction_filter/batch/encryption/EncryptUtil.java

```diff
@@ -238,7 +240,7 @@ public static void encryptFile(
             System.err.println(e);
```

Comment:
Should it be removed?

Reply:
That line has been fixed in this [PR](https://github.com/pagopa/rtd-ms-transaction-filter/pull/236)